tests/test_utils.py

```diff
@@ -732,6 +732,8 @@ def clean_athena_db(connection_string: str, db_name: str, table_to_keep: str) ->
         engine.dispose()
 
 
+# Chetan - 20220426 - This is to be deleted in favor of
+# `set_consistent_seed_within_numeric_metric_range_multi_batch_parameter_builder`
 def set_bootstrap_random_seed_variable(
     profiler: BaseRuleBasedProfiler,
     random_seed: int = RANDOM_SEED,
```

Comment:
Curious. Is this still used? If not, can it/ should it be deleted as part of this PR?

Reply:
It is! it's used in the volume data assistant tests - I'm keeping it here because it actually updates the underlying config (while `monkeypatch` does not).

We have a very extensive test that asserts expected against actual config and removing this causes it to break. Should be a simple enough fix but I didn't feel like reasoning about a thousand line diff 😬 

Both utilities are nice to have tbh